src/config/type-orm-config-test.service.ts

```diff
@@ -0,0 +1,15 @@
+import {Injectable} from '@nestjs/common';
+import {TypeOrmModuleOptions, TypeOrmOptionsFactory} from '@nestjs/typeorm';
+
+@Injectable()
+export class TypeOrmConfigTestService implements TypeOrmOptionsFactory {
+
+    createTypeOrmOptions(): TypeOrmModuleOptions {
+        return {
+            type: 'sqlite',
+            database: process.env.DB_NAME || 'kudos',
```

Comment:
Jeżeli dobrze rozumiem, to `ConfigService` powinien odpowiadać za pobieranie danych z enva. Może warto byłoby go tu użyć? A do tego mógłbyś przenieść domyślne wartości w tamto miejsce.

Reply:
Niestety próba wstrzyknięcia serwisu `ConfigService` w serwis `TypeOrmConfigService` kończyła sie niepowodzeniem i aplikacja mi w ogóle nie wstawała. Teraz skoro mam chwilę zobaczę czy da się coś z tym jeszcze zrobić.